mne/io/edf/edf.py

```diff
@@ -415,6 +421,13 @@ def _read_ch(fid, subtype, samp, dtype_byte, dtype=None):
 def _get_info(fname, stim_channel, annot, annotmap, eog, misc, exclude,
               preload):
     """Extract all the information from the EDF+, BDF or GDF file."""
+    # backward compat aliasing; code below wants to see None but in 0.18
+    # we allow/prefer False for consistency with BV/EEGLAB
+    stim_channel = None if stim_channel is False else stim_channel
+    if stim_channel == '':
```

Comment:
I'm not sure I understand this. An empty string evaluates to False, correct? So it will be None at this stage?

Reply:
It is true that `bool('') is False` but that doesn't apply here:
```
>>> '' == None
False
>>> '' == False
False
```
So it should behave as expected. The beauty of built-in types

Reply:
... and in this particular case (which is actually more strict than `==` anyway):
```
>>> '' is False
False
```